frontend/src/app/Settings/Facility/FacilityForm.test.tsx

```diff
@@ -214,9 +214,9 @@ describe("FacilityForm", () => {
         exact: false,
       })[0];
       await act(async () => await userEvent.clear(facilityZipCodeInput));
-      await act(async () => await userEvent.tab());
+      await clickSaveButton();
       const facilityZipCodeWarning = await screen.findByText(
-        "Facility zip code is missing"
+        "Facility ZIP code is required"
```

Comment:
similar note to the comment [here](https://github.com/CDCgov/prime-simplereport/pull/5983/files#r1235287974)